src/python/pants/backend/codegen/protobuf/target_types.py

```diff
@@ -60,7 +61,9 @@ class ProtobufSourceTarget(Target):
         ProtobufDependenciesField,
         ProtobufSourceField,
         ProtobufGrpcToggleField,
+        JvmJdkField,
```

Comment:
Rather than being declared on the generic target, this field should be installed as a plugin field, similar to: https://github.com/pantsbuild/pants/blob/cd44dd1ffc5671a4e7fb1d488f5fa9b0612ca262/src/python/pants/backend/docker/lint/hadolint/skip_field.py#L15

Reply:
And probably by subclassing `JvmJdkField` and then setting `alias = "jvm_jdk"` in the subclass.

Reply:
I didn't know about that one! Do we have that concept for `moved_fields` etc too?

Reply:
All plugin fields end up getting copied, right @stuhood? Will that be fine for parametrization of `protobuf_sources` (plural)?

Reply:
We'll probably need a solution for copied-vs-moved for plugin fields soon, yea. But still the right thing to use for now.

Reply:
Checks out. Thanks!